bindings/src/main/java/io/pravega/storage/hdfs/HDFSSimpleStorageFactory.java

```diff
@@ -13,18 +13,23 @@
 import io.pravega.segmentstore.storage.StorageFactory;
 import io.pravega.segmentstore.storage.chunklayer.ChunkedSegmentStorage;
 import io.pravega.segmentstore.storage.chunklayer.ChunkedSegmentStorageConfig;
+import lombok.AllArgsConstructor;
 import lombok.NonNull;
-import lombok.RequiredArgsConstructor;
 
 import java.util.concurrent.Executor;
 
 /**
  * Factory for HDFS {@link Storage} implemented using {@link ChunkedSegmentStorage} and {@link HDFSChunkStorage}.
  */
-@RequiredArgsConstructor
+@AllArgsConstructor
```

Comment:
I don't see why this is needed. RequiredArgsConstructor will generate the constructor for all final members.

Reply:
fixed